app/views/businesses/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<section class="user-input-container">
+  <header class="input-container">
+    <nav>
+      <ul class='links'>
+        <li>
+        <%= link_to categories_path do %>
+          <i class="fa fa-arrow-left"></i>
+          <% end %>
+        </li>
+        <li>
+          ADD TRANSACTION
+        </li>
+        <li class='login'>
+          <input type="submit" value='Add' form='form' class='submit'>
```

Comment:
- [ ] In the "Add a new transaction" page, the user should be able to select more than one category to create a transaction. As I see on your form, the user can only choose one category in the form. Please remember that the relationship between category and transaction is a many-to-many relationship.